src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

Comment:
The first link is for static maps.  😉 For Geocoding it looks the same. So yes, as both standard and premium users now have 50 requests per second, we shouldn’t limit it. We have a catch for `OVER_QUERY_LIMIT` somewhere. But I just saw that it doesn’t try again in that case. Maybe we should add that.

But: 50 per second means we can make a request every 20ms, not every 100ms.